german/schwimmen/insel/fs_aussen/fs_aussen.md

```diff
@@ -0,0 +1,12 @@
+Du gehst langsam auf den Fahrstuhl zu. Aus dem Nichts heraus wird er sich ja wohl nicht materialisiert 
+haben... oder etwa doch?
+Als du unmittelbar vor dem metallischen Rahmen der Fahrstuhltür stehst, fällt dir auf, daß allem Anschein nach
+diese Erscheinung nur eine Vorderseite, aber keine Seitenwände hat, denn unmittelbar neben dem Rahmen kannst du 
+ein paar Meter in den Wald hineinschauen ohne die Seite oder gar die Rückwand des Aufzugs sehen zu können.
+
+Vorsichtig berührst du den Rahmen der linken Schiebetür: er ist solide und fühlt sich kühl und glattpoliert an. 
+Als du deine Hand wieder zurückziehst, streifst du unabsichtlich den Zweig eines mittelhohen Strauchs - oder genauer gesagt: gleitet deine Hand vollkommen berührungslos durch eben diesen Zweig samt seiner Blätter hindurch...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.